src/components/auth/login/Login.js

```diff
@@ -68,7 +68,11 @@ export default class Login extends Component {
       return <Redirect to="/dashboard" />;
     } else {
       let errorDiv = '';
-      if (this.state.attemptedSubmit && !this.userInfoRetrieved()) {
+      if (
+        this.state.attemptedSubmit &&
+        !this.props.isAuthenticated &&
+        !this.props.isFetching
+      ) {
```

Comment:
Excellent, this fixes everything 😊

Reply:
Yupp!! No more error as you wait for it to fetch data lol